### PR TITLE
Gh1378

### PR DESCRIPTION
**Pull request by davidcarlisle**
# Internal housekeeping

## Status of pull request

- Ready to merge

## Checklist of required changes before merge will be approved
- [X] Test file(s) added
- [X] Version and date string updated in changed source files
- [X] Relevant `\changes` entries in source included
- [X] Relevant `changes.txt` updated
- [n/a]  Rollback provided (if necessary)? N/A
- [n/a] `ltnewsX.tex` (and/or `latexchanges.tex`) updated N/A

### CODE REVIEW

File: required/tools/testfiles/github-1378.lvt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+\documentclass{article}
+\usepackage{array}
+\input{test2e}
+\begin{document}
+
+x $x$
+
+\START
+% The error message shoul dshow spaces
```
Comment (muzimuzhi) [inline required/tools/testfiles/github-1378.lvt:9]: Also move 'd' :)
```suggestion
% The error message should show spaces
```

**Review by muzimuzhi — CHANGES_REQUESTED** [review]
minor typo

**Review by FrankMittelbach — APPROVED** [review]
with the suggested correction, fine

File: required/tools/testfiles/github-1378.lvt (hunk 1 above)
Comment (davidcarlisle) [reply required/tools/testfiles/github-1378.lvt:9]: sorry, will fix @muzimuzhi

**Review by josephwright — APPROVED** [review]

**Pull request merged**